inference-engine/tests/ie_test_utils/common_test_utils/ngraph_test_utils.cpp

```diff
@@ -1,33 +1,39 @@
-// Copyright (C) 2020 Intel Corporation
+// Copyright (C) 2020-2021 Intel Corporation
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include <string>
+#include "ngraph_test_utils.hpp"
+
+#include <cassert>
 #include <memory>
 #include <queue>
-#include <assert.h>
+#include <string>
 
 #include <ngraph/function.hpp>
-#include <ngraph/opsets/opset1.hpp>
 #include <ngraph/op/util/op_types.hpp>
+#include <ngraph/opsets/opset1.hpp>
 #include <ngraph/pass/visualize_tree.hpp>
-#include "ngraph_test_utils.hpp"
 
+namespace {
 bool isTypeRelaxed(const std::string& type) {
     return type.find_first_of("TypeRelaxed") == 0;
 }
 
 bool compareTypeInfo(const ngraph::DiscreteTypeInfo& info1, const ngraph::DiscreteTypeInfo& info2) {
-    if (!isTypeRelaxed(info1.name) && !isTypeRelaxed(info2.name) && (info1.version != info2.version)) {
+    if (!isTypeRelaxed(info1.name) && !isTypeRelaxed(info2.name) &&
+        (info1.version != info2.version)) {
         return false;
     }
 
-    const std::string info1Name = isTypeRelaxed(info1.name) && (info1.parent != nullptr) ? info1.parent->name : info1.name;
-    const std::string info2Name = isTypeRelaxed(info2.name) && (info2.parent != nullptr) ? info2.parent->name : info2.name;
+    const std::string info1Name =
+        isTypeRelaxed(info1.name) && (info1.parent != nullptr) ? info1.parent->name : info1.name;
+    const std::string info2Name =
+        isTypeRelaxed(info2.name) && (info2.parent != nullptr) ? info2.parent->name : info2.name;
     return info1Name == info2Name;
 }
 
-bool compare_rt_keys(const std::shared_ptr<ngraph::Node>& node1, const std::shared_ptr<ngraph::Node>& node2) {
+template <typename Node>
```

Comment:
Why did you add a template argument for this function? I don't see that we use it for different types.

Reply:
I add it because I thought I replace queue of `shared_ptr` to raw ptr inside `compare_function`. This change should not bring any change after compilation. It's hidden in unnamed namespace and most probably it will be inlined.